src/index.ts

```diff
@@ -67,10 +69,22 @@ const listrTasks = [
 
   // Linting
   eslint,
+  ...(!(
+    '@upleveled/react-scripts' in projectPackageJson.dependencies! ||
+    'next' in projectPackageJson.dependencies!
+  )
+    ? []
+    : [stylelint]),
   prettier,
 
   // Version and configuration checks
   eslintConfigIsValid,
+  ...(!(
+    '@upleveled/react-scripts' in projectPackageJson.dependencies! ||
```

Comment:
instead of using the `!` here you'll probably need to check whether `dependencies` exists (like in the condition above)

otherwise you get a crash:

<img width="548" alt="Screenshot 2023-04-21 at 11 39 33" src="https://user-images.githubusercontent.com/1935696/233603383-4970c849-53b5-451a-b315-12532041af06.png">

Reply:
778935c

following a similar approach as https://github.com/upleveled/eslint-config-upleveled/blob/99c14d946b1f52f7d37a32a3b4ac83b6aa3fcb0b/bin/install.js#L20